src/main/java/com/flightstats/hub/app/HubMain.java

```diff
@@ -57,13 +61,15 @@
 public class HubMain {
 
     private static final DateTime startTime = new DateTime();
-    private final StorageBackend storageBackend = StorageBackend.valueOf(HubProperties.getProperty("hub.type", "aws"));
+    private static PropertiesLoader propertiesLoader = PropertiesLoader.getInstance();
+    private final StorageBackend storageBackend = StorageBackend.valueOf(propertiesLoader.getProperty("hub.type", "aws"));
```

Comment:
I would like to see the usage of "hub.type" refactored to not reference aws but be something more like single, clustered, encrypted, readOnly, etc... and have the aws designation be called app.storage.backend or something. But that is definitely a separate PR